lib/views/create-bug.js

```diff
@@ -225,7 +220,8 @@ CreateBug.prototype.render = function(ctx) {
     // Activity data overrides locally stored data for now
     var activity = self.app.activity;
     if (activity) {
-      addAttachments(activity.data.blobs, activity.data.filenames);
+      addActivityAttachmentsAttachments(activity.data.blobs,
```

Comment:
Heh autoComplete fail, that was one of those "everything works lets just make the function name a bit clearer and it obviously wont break anything" changes